build/post-configure.sh.in

```diff
@@ -50,6 +50,7 @@ echo "Enable built-in BLAS implementation............. : @fla_enable_builtin_bla
 echo ""
 echo "Enable lapack2flame............................. : @fla_enable_lapack2flame@"
 echo "Enable lapack2flash (experimental).............. : @fla_enable_lapack2flash@"
```

Comment:
Can you add documentation for the new legacy LAPACK option?

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/4/commits/6c46c7ba244bcde2c0c4896f2cd9ed9b70aee18b